### ISSUE TITLE
Keyboard navigation in `DataTable` is only available after a cell is clicked with the mouse

### ISSUE THREAD

**Issue by apcamargo** [Textualize/textual#1345]
In a data table app, keyboard navigation doesn't work until you click a cell with the mouse. This can be reproduced in the [`DataTable` example ](https://textual.textualize.io/widgets/data_table/#__tabbed_1_2)in the documentation.

I'm using iTerm 3.4.18, Textual 0.6.0.

**Comment by davep** [comment]
The issue isn't that you need to click with a mouse, it's that, as with any focusable widget in a Textual application, the widget needs to have focus before it can receive input. By default no widget will have focus. So, with the `DataTable` example, if you don't touch the mouse but instead press the <kbd>tab</kbd> key and then navigate with the keyboard you'll find you can move around.

More commonly, if you wish a widget to have focus on startup of your application, you'd set that focus in `on_mount` of your app (or main screen, depending on your approach). You can test this by adding this:

```python
table.focus()
```

to the `on_mount` in the example you mention.

**Comment by apcamargo** [comment]
Ok, got it. Thanks for the clarification.
